curriculum/utils.js

```diff
@@ -16,3 +16,32 @@ exports.testedLang = function testedLang() {
     throw Error('LOCALE must be set for testing');
   }
 };
+
+// TODO: migrate to TS and use the SuperBlocks enum from
+// config/certification-settings.ts
+
+const superBlockToOrder = {
+  'responsive-web-design': 0,
+  'javascript-algorithms-and-data-structures': 1,
+  'front-end-development-libraries': 2,
+  'data-visualization': 3,
+  'relational-databases': 4,
+  'back-end-development-and-apis': 5,
+  'quality-assurance': 6,
+  'scientific-computing-with-python': 7,
+  'data-analysis-with-python': 8,
+  'information-security': 9,
+  'machine-learning-with-python': 10,
+  'coding-interview-prep': 11
+};
+
+function getSuperOrder(superblock, { isLegacy } = { isLegacy: false }) {
+  if (typeof superblock !== 'string')
+    throw Error('superblock must be a string');
+  const order = superBlockToOrder[superblock];
+  if (typeof order === 'undefined')
+    throw Error(`${superblock} is not a valid superblock`);
```

Comment:
This is throwing on the `certifications` superblock.

Reply:
Hmm.  Do we want to give that a `null` `superOrder`?

Reply:
Or filter the test to not care about the `certifications` superblock.

Reply:
Could do.  Since `superOrder` doesn't really mean anything for `certifications` probably safest to never set it, though.